ruby-gem/lib/calabash-android/operations.rb

```diff
@@ -618,7 +618,7 @@ def wake_up
         log wake_up_cmd
         raise "Could not wake up the device" unless system(wake_up_cmd)
 
-        Retriable.retriable :tries => 10, :interval => 1 do
+        Retriable.retriable :tries => 10, :base_interval => 1 do
```

Comment:
`tries` and `base_interval` work as desired, according to the retrieable documentation:

```
tries (default: 3) - Number of attempts to make at running your code block (includes intial attempt).

base_interval (default: 0.5) - The initial interval in seconds between tries.
```